saleor/static/js_src/dashboard.jsx

```diff
@@ -0,0 +1,78 @@
+var SORT_INFO = [ { name: 'id', dir: 'asc'}];
+
+class DataTable extends React.Component {
+    state = {
+        data: this.props.data
+    };
+
+    handleSortChange(sortInfo) {
+        SORT_INFO = sortInfo;
+
+        this.setState({data: sort(this.props.data)});
+    };
+
+    handleColumnOrderChange(index, dropIndex) {
+        var col = columns[index];
+        columns.splice(index, 1); //delete from index, 1 item
+        columns.splice(dropIndex, 0, col);
+        this.setState({})
+    };
+
+    render() {
+        return <DataGrid
+            idProperty={this.props.columns[0]}
+            dataSource={this.state.data}
+            columns={this.props.columns}
+            sortInfo={SORT_INFO}
+            onSortChange={this.handleSortChange.bind(this)}
```

Comment:
Could be `{::this.handleSortChange}` or we could import the [`autobind-decorator`](https://github.com/andreypopp/autobind-decorator) decorator package.